contracts/contracts/utilities/Utilities.sol

```diff
@@ -19,9 +24,32 @@ contract CommonUtilities {
   }
 }
 
-contract Utilities is SnarkConstants, Hasher, IPubKey, IMessage {
+/// @title Utilities
+/// @author PSE
+/// @notice An utility contract that can be used to:
+/// hash a state leaf
+/// pad and hash a MACI message
+/// hash a MACI message and an encryption public key
```

Comment:
I guess you should put commons at the end of each sentence, else they will be presented connectedly.

Reply:
/// * hash a state leaf  
/// * pad and hash a MACI message  
/// * hash a MACI message and an encryption public key

Looks like I can make them as a list like this :D - how's it looking now?

Reply:
LGTM